### PR TITLE
update rewards + slashing

### PR DESCRIPTION
**Pull request by junkicide**

### CODE REVIEW

File: src/architecture/economics/proof-of-stake/reward-distribution.md [hunk 1]
```diff
@@ -160,8 +160,8 @@ This can be implemented as a negative inflation rate for a particular block.
 
 Instant redelegation is not supported. Redelegations must wait the unbonding period.
 
-## State management
+<!--## State management
```
Comment (cwgoes) [inline src/architecture/economics/proof-of-stake/reward-distribution.md:163]: Do we no longer do this?

File: src/architecture/economics/proof-of-stake/cubic-slashing.md [hunk 2]
```diff
@@ -4,27 +4,27 @@ Namada implements cubic slashing, meaning that the amount of a slash is proporti
...
-  // then exponential between 0 & 1/3 voting power
-  // we can make this a more complex function later
-}
+
+```haskell =
+calculateSlashRate :: [Slash] -> Float
+
+calculateSlashRate slashes = 
+    let votingPowerFraction = sum [ votingPowerFraction (validator slash) | slash <- slashes]
+	in max(0.01, min(1, votingPowerFraction**2 * 9))
```
Comment (cwgoes) [inline src/architecture/economics/proof-of-stake/cubic-slashing.md:20]: Haskell does not need parentheses

**Review by cwgoes — CHANGES_REQUESTED** [review]
Two minor nits, otherwise LGTM

File: src/architecture/economics/proof-of-stake/reward-distribution.md (hunk 1 above)
Comment (junkicide) [reply src/architecture/economics/proof-of-stake/reward-distribution.md:163]: indeed

**Pull request merged**